app/scripts/modules/core/src/projects/dashboard/pipeline/ProjectPipeline.tsx

```diff
@@ -53,6 +53,7 @@ export class ProjectPipeline extends React.Component<IProjectPipelineProps, IPro
     const execution = this.props.execution;
     const stages = execution.stageSummaries.map((stage) => (
       <ExecutionMarker
+        manualJudgment={[]}
```

Comment:
Why is this defaulted to an empty array here? Would it make more sense to make this an optional prop or default it to an empty array within the component?